src/components/atomic/RemovableTag/__tests__/Removable.tests.js

```diff
@@ -0,0 +1,19 @@
+import { fireEvent, render } from "@testing-library/react";
```

Comment:
why is this filename not `RemovableTag.tests.js`?

Reply:
Thanks for the catch, that was an omission. Will update that.

Reply:
Done.